README.md

```diff
@@ -86,18 +86,32 @@ It makes an executable in `pkg/`, e.g. `pkg/digdag-$VERSION.jar`.
 ### Releasing a new version
 
 You need to set Bintray user name and API key in `BINTRAY_USER` and `BINTRAY_KEY` environment variables.
-
-1. run `git pull origin --tags`.
-2. run `./gradlew setVersion -Pto=<version>` command.
-3. write release notes to `releases/release-<version>.rst` file. It must include at least version (the first line) and release date (the last line).
-4. run `./gradlew clean cli site check releaseCheck`.
-5. if it succeeded, run `./gradlew release`.
-6. create next snapshot version, run `./gradlew setVersion -Pto=<next-version>-SNAPSHOT`.
-7. push to master.
-8. create a release in [GitHub releases](https://github.com/treasure-data/digdag/releases).
+In the following instructions, assumed that `upstream` is set to `treasure-data/digdag` and `origin` is set to your private repository.
+
+1. run `git pull upstream --tags`.
+1. run `./gradlew setVersion -Pto=<version>` command.
+1. write release notes to `releases/release-<version>.rst` file. It must include at least version (the first line) and release date (the last line).
+1. run `./gradlew clean cli site check releaseCheck`.
+1. make a release branch. `git checkout -b release_v<version>` and commit. 
+1. push the release branch to origin and create a PR.
+1. after the PR is merged to master, checkout master and pull latest upstream/master.
+1. run `./gradlew clean cli site check releaseCheck` agian.
+1. if it succeeded, run `./gradlew release`.
+1. a few minutes later, run `digdag selfupdate` and confirm the version.
```

Comment:
the numbering of items looks good?

Reply:
@muga Thanks for your pointing out.
In my understand, markdown automatically make the number in ordered list and we don't need to take care of the written number.

https://github.com/treasure-data/digdag/blob/144f121e5b99a6fec49105cbf5c1704970e5c385/README.md

If we view the file as above. List is correctly numbered.
But if this may be confused, I will change it.

Reply:
Ah! got it. thank you for explaining it. i didn't know that..